test/uk/gov/hmrc/pillar2/service/SubscriptionServiceSpec.scala

```diff
@@ -89,6 +89,44 @@ class SubscriptionServiceSpec extends BaseSpec with Generators with ScalaCheckPr
       }
 
     }
+
+    "handle LimitedLiabilityPartnership entity type correctly" - {
+      "when all required data is present" in {
+        when(
+          mockSubscriptionConnector
+            .sendCreateSubscriptionInformation(any[RequestDetail]())(any[HeaderCarrier](), any[ExecutionContext]())
+        ).thenReturn(
+          Future.successful(
+            HttpResponse.apply(OK, "Success")
+          )
+        )
+
+        forAll(arbitrary[String], Gen.option(arbitrary[String]), arbitraryWithIdUpeFmUserDataLLPUserAnswers.arbitrary) {
+          (upeSafeId, fmSafeId, userAnswers) =>
+            service.sendCreateSubscription(upeSafeId, fmSafeId, userAnswers).map { response =>
+              response.status mustBe OK
+            }
+        }
+      }
+
+      // "throw exception when company profile is missing" in {
```

Comment:
do we need this?

Reply:
Ah yeah, probably.  apologies did this on friday and just put up the PR today so I kinda forgot about this - will update